src/main/java/YC.java

```diff
@@ -0,0 +1,69 @@
+import java.util.Scanner;
+public class YC {
+    public static void main(String[] args) {
+        String bot_Name = "YC";
```

Comment:
also consider private static final if constant
```suggestion
       private static final String BOT_NAME + "YC";
       ```